federatedscope/gfl/model/model_builder.py

```diff
@@ -71,7 +71,7 @@ def get_gnn(model_config, local_data):
                              max_depth=model_config.layer,
                              dropout=model_config.dropout,
                              gnn=model_config.type)
-    elif model_config.task == 'graph':
+    elif model_config.task.startswith('graph'):
```

Comment:
We'd better modify `node` and `link` too.

Reply:
Modified accordingly